client/app/cloud/project/compute/loadbalancer/cloud-project-compute-loadbalancer.service.js

```diff
@@ -0,0 +1,103 @@
+class CloudProjectComputeLoadbalancerService {
+    constructor ($q, OvhApiCloudProjectIplb, OvhApiIpLoadBalancing) {
+        this.$q = $q;
+        this.OvhApiCloudProjectIplb = OvhApiCloudProjectIplb;
+        this.OvhApiIpLoadBalancing = OvhApiIpLoadBalancing;
+    }
+
+    getLoadbalancer (id) {
+        // Get loadbalancer
+        return this.OvhApiIpLoadBalancing.Lexi().get({
+            serviceName: id
+        }).$promise.then(loadbalancer => {
+            if (loadbalancer.state !== "ok") {
+                return loadbalancer;
+            }
+            // Find the frontend http 80 if exists
+            return this.OvhApiIpLoadBalancing.Frontend().Http().Lexi().query({
+                serviceName: id,
+                port: 80
+            }).$promise.then(frontendIds =>
+            // Get frontend details
+                frontendIds.length && this.OvhApiIpLoadBalancing.Frontend().Http().Lexi().get({
+                    serviceName: id,
+                    frontendId: frontendIds[0]
+                }).$promise || loadbalancer
+            ).then(frontend => {
+                if (frontend.frontendId) {
+                    loadbalancer.frontend = frontend;
+                }
+                // Get default farm details
+                return frontend.frontendId && frontend.defaultFarmId && this.OvhApiIpLoadBalancing.Farm().Http().Lexi().get({
+                    serviceName: id,
+                    farmId: frontend.defaultFarmId
+                }).$promise || loadbalancer;
+            }).then(farm => {
+                if (farm.farmId) {
+                    loadbalancer.farm = farm;
+                }
+                return loadbalancer;
+            });
+        }
+        ).then(loadbalancer => {
```

Comment:
put }) on same line.

Also, we prefer => 
getSomePromise()
    .then()
    .catch()
    .finally